docs/sources/enterprise/access-control/about-rbac.md

```diff
@@ -53,13 +51,13 @@ Each basic role is comprised of a number of _permissions_. For example, the view
 You can use RBAC to modify the permissions associated with any basic role, which changes what viewers, editors, or admins can do. For more information about the permissions associated with each basic role, refer to [Basic role definitions]({{< relref "./rbac-fixed-basic-role-definitions#basic-role-assignments" >}}).
 You cannot delete basic roles.
 
-> **Note:** You must assign each Grafana user a basic role.
+> **Note:** You can't have a Grafana user without a basic role assigned.
```

Comment:
Why change this language? It's a double-negative. Is there something incorrect about the original note here?

Reply:
Thanks for the question, really good point. Grafana by default assigns a basic role to a user, when I read "must assign", it made an impression that I have to perform an action - whereas users don't need to perform any action. The note however I found useful to make, since we are eventually going to remove this limitation and allow users to not have any basic role assigned - this is something we have also received as a feedback from customers. 

I don't have any strong opinion to be honest, ultimately I think if we remove the note entirely, that should also be fine.

Reply:
Gotcha, yeah, feel free to remove the note.